src/main/java/org/broadinstitute/hellbender/tools/gvs/common/CommonCode.java

```diff
@@ -44,9 +44,10 @@ public static List<Integer> getGTAlleleIndexes(final VariantContext variant) {
 
     public static String getGTString(final VariantContext variant) {
         List<Integer> allele_indices = getGTAlleleIndexes(variant);
-        if (allele_indices.size() != 2){
-            throw new IllegalArgumentException("GT doesnt have two alleles");
-        }
+
+        // As of VS-910, we aren't going to consider any number of alleles to necessarily be a UserException
+        // This code used to throw an error if allele_indices.size() != 2
```

Comment:
I do think in the future we can add validation that checks to see if a variant with allele_indices.size() != 2 is specifically on chrX or chrY if this ends up being something that we want

Reply:
agreed.  If/when we can learn more about to what degree we are expected to support different ploidies (and if we'll only accept them in some places) we can put some constraints in here.  It's why I wanted to leave comments behind showing where we _used_ to care, in case we want to shrink down our range of permissibility again in the future.